tests/unit/test_ps_model.py

```diff
@@ -0,0 +1,191 @@
+    np.testing.assert_array_almost_equal(engine_efficiency, np.array([0.315, 0.316]), decimal=3)
+
+    # Test fuel mass flow rate
+    fuel_flow = ps.fuel_mass_flow_rate(
+        air_pressure, air_temperature, mach_number, c_t, engine_efficiency,
+        atyp_param.wing_surface_area, q_fuel=43e6
+    )
+
+    fuel_flow = ps.correct_fuel_flow(
+        fuel_flow, altitude_ft, air_temperature, air_pressure, mach_number,
+        atyp_param.ff_idle_sls, atyp_param.ff_max_sls
+    )
+    np.testing.assert_array_almost_equal(fuel_flow, np.array([0.574, 0.559]), decimal=3)
+
+
+def test_normalised_aircraft_performance_curves():
+    """
+    For a given Mach number, there is a pair of overall propulsion efficiency (ETA) and thrust coefficient (c_t) at
+    which the ETA is at its maximum. A plot of `eta_over_eta_b` over `c_t_over_c_t_eta_b` should be an inverse
+    U-shape and the global maximum of `eta_over_eta_b` should occur where `c_t_over_c_t_eta_b` is equal to 1.
+    """
+    aircraft_type_icao = "A320"
+    f_thrust = np.arange(10000, 60000, 500)
+    mach_num = np.ones_like(f_thrust) * 0.750
+    altitude_ft = np.ones_like(f_thrust) * 40000
+    air_pressure = ft_to_pl(altitude_ft) * 100
+
+    # Extract aircraft properties for aircraft type
+    ps_model = ps.PSModel()
+    atyp_param = ps_model.aircraft_engine_params[aircraft_type_icao]
+    mach_num_design_opt = atyp_param.m_des
+
+    # Derived coefficients
+    c_t = ps.engine_thrust_coefficient(f_thrust, mach_num, air_pressure, atyp_param.wing_surface_area)
+    c_t_eta_b = ps.max_thrust_coefficient(mach_num, atyp_param.m_des, atyp_param.c_t_des)
+    c_t_over_c_t_eta_b = c_t / c_t_eta_b
+
+    eta = ps.overall_propulsion_efficiency(mach_num, c_t, atyp_param)
+    eta_b = ps.max_overall_propulsion_efficiency(
+        mach_num_design_opt, mach_num_design_opt, atyp_param.eta_1, atyp_param.eta_2
+    )
+    eta_over_eta_b = eta / eta_b
+
+    i_max = np.argmax(eta_over_eta_b)
+
+    # Global maximum of `eta_over_eta_b` should occur where `c_t_over_c_t_eta_b` is equal to 1
+    assert (c_t_over_c_t_eta_b[i_max] > 0.99) and (c_t_over_c_t_eta_b[i_max] < 1.01)
+
+
+def test_total_fuel_burn():
+    df_flight = pd.read_csv(get_static_path("flight.csv"))
+    flight = Flight(df_flight.iloc[:100])
+
+    # add parameters
+    flight.attrs.update({"flight_id": "1", "aircraft_type": "A320"})
+
+    # Aircraft performance model
+    ps_model = ps.PSModel()
+    aircraft_performance = ps_model.simulate_fuel_and_performance(
+        aircraft_type_icao=flight.attrs["aircraft_type"],
+        altitude_ft=flight.altitude_ft,
+        time=flight["time"],
+        true_airspeed=knots_to_m_per_s(flight["speed"]),
+        air_temperature=m_to_T_isa(flight["altitude"]),
+        q_fuel=43.2e6,
+        correct_fuel_flow=False,
+        aircraft_mass=None
+    )
+    np.testing.assert_array_almost_equal(np.nansum(aircraft_performance.fuel_burn), 3805.97, decimal=1)
```

Comment:
We're just working with python floats here, so this could be clear with
```python
assert np.nansum(aircraft_performance.fuel_burn) == pytest.approx(3805.97, abs=0.1)
```